# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/doc/Debugger/Debugger.Frame.md
# Commit: 31d14cf1975c
# Full Hash: 31d14cf1975c1b23ef3db176252f8cc3b31a0772
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 2
# Description:
#   Bug 1600204 - Part 7: Support .this on suspended generator frames. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55580
# ==============================================================================

diff -r fa46d8caefde -r 31d14cf1975c js/src/doc/Debugger/Debugger.Frame.md
--- a/js/src/doc/Debugger/Debugger.Frame.md	Sun Dec 08 23:54:15 2019 +0000
+++ b/js/src/doc/Debugger/Debugger.Frame.md	Sun Dec 08 23:54:38 2019 +0000
@@ -201,7 +201,7 @@
 The value of `this` for this frame (a debuggee value). For a `wasmcall`
 frame, this property throws a `TypeError`.
 
-Accessing this property will throw if `.onStack == false`.
+Accessing this property will throw if `.terminated == true`.
 
 ### `older`
 The next-older visible frame, in which control will resume when this